docgen/parser/parser.go

```diff
@@ -463,14 +463,16 @@ func generateResponseExample(messages map[string]*Message, name string) (string,
 // normalizeRequestFields is used to filter out all fields marked with an predefined indicator of a request message.
 func normalizeRequestFields(messages map[string]*Message, name string) {
 	const hidingIndicator = "docgen: hide"
-	var filtered []*Field
-	for _, f := range messages[name].Fields {
-		if strings.Contains(f.Comment.Leading, hidingIndicator) {
-			continue
+	for _, message := range messages {
+		var filtered []*Field
+		for _, f := range message.Fields {
+			if strings.Contains(f.Comment.Leading, hidingIndicator) {
+				continue
+			}
+			filtered = append(filtered, f)
```

Comment:
For this you could try reformatting as 
```
for _, f := range message.Fields {
			if !strings.Contains(f.Comment.Leading, hidingIndicator) {
				filtered = append(filtered, f)
			}
		}

```
to avoid use of continue, but it is just a suggestion, this is fine too 🙂